aQute.libg/src/aQute/lib/io/FileTree.java

```diff
@@ -88,23 +88,20 @@ public void addExcludes(List<String> addExcludes) {
 	 */
 	public List<File> getFiles(File baseDir, List<String> defaultIncludes) throws IOException {
 
-		List<Pattern> defaultPatternList = new ArrayList<>();
-		if (defaultIncludes != null) {
-			for (String defaultInclude : defaultIncludes) {
-				defaultPatternList.add(AntGlob.toPattern(defaultInclude));
-			}
-		}
 
-		List<Pattern> includePatterns = includes.isEmpty() && files.isEmpty() && (!defaultPatternList.isEmpty())
-			? defaultPatternList
+		List<Pattern> includePatterns = includes.isEmpty() && files.isEmpty() && (!defaultIncludes.isEmpty())
```

Comment:
You don't support defaultIncludes as null. Just do a null check here. If empty, then includes and files are also empty, so the true path of the expression will make an empty list which is OK.